package.json

```diff
@@ -148,6 +148,7 @@
         "node-fetch": "^2.6.1",
         "nyc": "^15.1.0",
         "prettier": "2.4.1",
+        "pyright": "^1.1.184",
```

Comment:
Whoa, it's slightly wild to me that Python tooling is distributed via npm 😂

Reply:
Ya it is funny. It looks like they built it in Typescript for fast & easy VSCode integration, since that was their first priority.